roles/kafka-ws/templates/kafkaws_deployment.yaml.j2

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: kafkaws
+    app: dojot
+  name: kafkaws
+  namespace: {{ dojot_namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dojot
+      name: kafkaws
+  template:
+    metadata:
+      labels:
+        name: kafkaws
+        app: dojot
+    spec:
+      containers:
+      - image: dojot/kafka-ws:{{ dojot_kafkaws_version }}
+        name: kafkaws
+        env:
+        - name: KAFKA_HOSTS
+          value: kafka-server:{{ dojot_kafka_port }}
+        - name: KAFKA_GROUP_ID
+          value: kafka-ws-group
+        - name: LOG_LEVEL
+          value: info
+        - name: LOG_VERBOSE
+          value: "false"
+        - name: KAFKA_WS_PORT
+          value: "5000"
+        - name: KAFKA_WS_JWT_EXP_TIME
+          value: "true"
+        - name: REDIS_HOST
+          value:  localhost
+      - name: kafkaws-redis
```

Comment:
Shouldn't Redis be separated from Kafka WS, so we could independently scale it?

Reply:
Maybe, i don't know. What @andersonluisribeiro @rascaraficci  think about that?

Reply:
I'm not sure, but I thinking about secure maybe It's better in the same deployment...  
Since in this case,  I don't need to create a service to expose the port from redis inside the cluster and we don't use TLS to communication between redis and kafka-ws... 
But I'm not sure, It's more a question than a affirmation.